test/influx.test.js

```diff
@@ -0,0 +1,58 @@
+import { describe, it, expect, vi } from 'vitest';
+import { reportMetric, createMetricsFromRequest } from '../lib/influx.js';
+
+describe('reportMetric', () => {
+  it('should report metrics correctly', async () => {
```

Comment:
```suggestion
  it('should write metrics to InfluxDB over HTTP', async () => {
```

We usually omit "should" from test names because it's too repetitive.

```suggestion
  it('writes metrics to InfluxDB over HTTP', async () => {
```